modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1023,6 +1025,29 @@ public String getTypeDeclaration(Schema p) {
         return super.getTypeDeclaration(target);
     }
 
+    private String getTypeDeclarationForArray(Schema<?> items) {
```

Comment:
Why do you call it "ForArray"? Isn't it valid for all Container types?

Reply:
The method is used only once.  For OpenAPI this is an array element.

```
if (ModelUtils.isArraySchema(target)) {
  ...
            String typeDeclaration = getTypeDeclarationForArray(items);
```

What name would you use?